### DISCUSSION TITLE
Beginner Help

### DISCUSSION THREAD

**Discussion by mdb17** [awawa-dev/HyperHDR#312] [Q&A] [answered]
So I have started playing with this project over the last few days. I had it up and running and controlling the LEDs behind the but couldn't get the USB grabber to display the video feed. So as a troubleshooting step I upgraded to the 18.0.0beta3 but now I can get the LEDs to turn on. 

I am running on a RPi3b driving the LEDs directly from the GIOP pin. I am not sure what steps I can take to start the trouble shooting process.

**Comment by awawa-dev** [comment]
Which LED strip type are you using? Please make sure you followed FAQ: https://github.com/awawa-dev/HyperHDR#faq

**Comment by mdb17** [comment]
I am using the WS281x. I was able to control the LEDs using 17.0.0. 

I am also having trouble loading the webpage, and saw this discussion: https://github.com/awawa-dev/HyperHDR/discussions/262#discussion-4039317 but since my pi image was downloaded 3 days ago I don't think it applies.

**Comment by awawa-dev** [comment]
The newer or older SD image doesn't matter... the root configuration is non standard setup and you must do it yourself after you flashed new SD image. The rest is in logs.

**Comment by awawa-dev** [comment]
From the FAQ:
![obraz](https://user-images.githubusercontent.com/69086569/183118252-caa605ae-00a9-4706-8136-fa6b94c8e20f.png)

**Reply by mdb17** [reply]
I saw that this morning, but completely missed the link. I am trying that now.

**Comment by awawa-dev** [comment]
```
sudo systemctl disable --now hyperhdr@pi
sudo systemctl enable --now hyperhdr@root
sudo reboot
```

**Comment by mdb17** [comment]
Thanks so much for the help so far. I apologize for missing that info that has gotten the t the lights working.

I am still not able to get the capture card working though. Here is a copy of my logs: https://pastebin.com/jpK8Ai6q

**Comment by awawa-dev** [comment, marked as answer]
```
2022-08-05T16:34:00.080Z [PERFORMANCE] [USB capturing: FPS = 10.00, decoding = 4ms, frames = 600, invalid = 0], [INSTANCE0: FPS = 10.00, processed = 600], [LED0: FPS = 4.00, send = 240, processed = 3000]
```

Hard to say. From logs the grabber is working just fine, at least it's sending video frames to HyperHDR. Maybe it's not capable to capture the video source and it's sending black frames. It may happen when HDCP comes in play or the input video stream has too high bandwidth for the grabber.

**Comment by mdb17** [comment]
That got me thinking, and when my source to 1080p instead of 4k it started working! Thanks again for all the help. It is really impressive what you have accomplished.
